storage/remote/queue_manager_test.go

```diff
@@ -98,7 +98,7 @@ func TestSampleDelivery(t *testing.T) {
 
 	c := NewTestStorageClient()
 	c.expectSamples(samples[:len(samples)/2])
-	m := NewStorageQueueManager(c, cfg)
+	m := NewStorageQueueManager(c, &cfg)
```

Comment:
...now you can get rid of `cfg` and just pass in `nil` here and in the other cases.